Makefile

```diff
@@ -0,0 +1,252 @@
+# Makefile for MCPJungle
```

Comment:
We don't need make file as discussed with @duaraghav8 
goreleaser actually makes sense without a ci here 

also this make files have tons of command , which is not necessary

Reply:
But it would be great if we just have 2 commands like
TODO

Build the MCPJungle UI:

```
cd ui
npm install
npm run build
```

Build the MCPJungle binary:

```
cd ..
make build
```